src/test/java/org/openmainframeproject/cobolcheck/ExpanderTest.java

```diff
@@ -66,7 +66,7 @@ void commonSetup() throws IOException {
         }
     }
 
-    @Test
+/*     @Test
```

Comment:
@samdion1994 
Would you please elaborate on why you need to remove this test ?
If it is no longer passing, then the test should be fixed, instead of removed.